declarative/src/callproxymodel.cpp

```diff
@@ -1,6 +1,7 @@
 #include "callproxymodel.h"
 #include "event.h"
 #include "eventmodel.h"
+#include "debug_p.h"
```

Comment:
debug.h on this side? Same on draftevent and mmshelper.

Reply:
Oh sorry. It seems I'm not properly awaken this morning.